sys-apps/edac-utils/edac-utils-0.18-r1.ebuild

```diff
@@ -19,9 +19,18 @@ RDEPEND="${DEPEND}
 	sys-apps/dmidecode"
```

Comment:
It's a few lines above but we drop [back](https://devmanual.gentoo.org/keywording/index.html) to ~amd64 when bumping usually.